app/pods/components/web-components/vl-table-actions/template.hbs

```diff
@@ -9,40 +9,25 @@
         {{action "openDocument" row bubbles=false}}
       >
         <i class="vl-button__icon ki-document"></i>
-        {{#attach-tooltip
-          arrow="true"
-          animation="shift"
-          placement="bottom"
-          class="ember-attacher-tooltip"
-        }}
+        <EmberTooltip @side="bottom" @tooltipClass="tooltip-custom">
           {{t "no-nota"}}
-        {{/attach-tooltip}}
+        </EmberTooltip>
       </button>
     {{else}}
       <button type="button" class="vl-button vl-button--link vl-button--icon"
         {{action "openDocument" row bubbles=false}}
       >
         <i class="vl-button__icon ki-document"></i>
-        {{#attach-tooltip
-          arrow="true"
-          animation="shift"
-          placement="bottom"
-          class="ember-attacher-tooltip"
-        }}
-          {{t "open-note"}}
-        {{/attach-tooltip}}
```

Comment:
"open-note" / "open-nota",  per ongeluk gewijzigd?

"open-nota" lijkt niet te bestaan in nl-be.json